examples/spec/lcdproc/LCDd.ini

```diff
@@ -0,0 +1,2752 @@
+[]
+mountpoint = LCDd.conf
+;infos/plugins = dump validation enum range ipaddr path network type
+infos/plugins = dump validation enum
+
+;Until the max plugin error is not fixed, some specifications will be left commented out
+
+;=============================================================================
+;=============================================================================
+
+;Affecting all subsettings, eg curses/Contrast, CFontz/Contrast, etc.
+;------------------------------------
+
+[/_/contrast]
+type = unsigned_short
+check/type = unsigned_short
+check/range = 0-1000
+description = Set the initial contrast
+
+[/_/brightness]
+type = unsigned_short
+check/type = unsigned_short
+check/range = 0-1000
+
+[/_/offbrightness]
+type = unsigned_short
+check/type = unsigned_short
+check/range = 0-1000
+
+[/_/reboot]
+type = boolean
+check/type = boolean
+default = no
+description = Reinitialize the LCD`s BIOS normally you shouldn`t need this
+
+;Server
+;----------------------------------------
+[/server/driverpath]
+type = string
+check/type = string
+check/validation = .*[\/]$
+check/validation/match = LINE
+check/validation/message = The path has to contain a backslack (/) at the end
+default = server/drivers/
+description = Select the LCD size
+
+
+[/server/driver]
+type = string
+check/type = string
+default = curses
+description = "Tells the server to load the given drivers. Multiple lines can be given.
+  The name of the driver is case sensitive and determines the section
+  where to look for further configuration options of the specific driver
+  as well as the name of the dynamic driver module to load at runtime.
+  The latter one can be changed by giving a File= directive in the
+  driver specific section.
+
+  The following drivers are supported:
+    bayrad, CFontz, CFontzPacket, curses, CwLnx, ea65, EyeboxOne, futaba,
+    g15, glcd, glcdlib, glk, hd44780, icp_a106, imon, imonlcd,, IOWarrior,
+    irman, joy, lb216, lcdm001, lcterm, linux_input, lirc, lis, MD8800,
+    mdm166a, ms6931, mtc_s16209x, MtxOrb, mx5000, NoritakeVFD,
+    Olimex_MOD_LCD1x9, picolcd, pyramid, rawserial, sdeclcd, sed1330,
+    sed1520, serialPOS, serialVFD, shuttleVFD, sli, stv5730, svga, t6963,
+    text, tyan, ula200, vlsys_m428, xosd, yard2LCD"
+
+[/server/bind]
+type = string
+;check/ipaddr =
```

Comment:
say why it is commented out

Reply:
From the Pull Request Description:

> Please note that some checks are intentionally commented out because of the max plugin error.
> With this they can at least still be spec-mounted.